runtimes/polimec/src/lib.rs

```diff
@@ -244,14 +244,8 @@ impl Contains<RuntimeCall> for BaseCallFilter {
 							pallet_funding::Call::end_evaluation { .. } |
 							pallet_funding::Call::evaluate { .. } |
 							pallet_funding::Call::start_auction { .. } |
-							pallet_funding::Call::root_do_auction_opening { .. } |
-							pallet_funding::Call::root_do_start_auction_closing { .. } |
 							pallet_funding::Call::bid { .. } |
-							pallet_funding::Call::root_do_end_auction { .. } |
 							pallet_funding::Call::contribute { .. } |
-							pallet_funding::Call::root_do_community_funding { .. } |
```

Comment:
Add the new runtime calls to the filter, so root -> non root call instead of only removing root?